Assets/Scripts/CameraManager.cs

```diff
@@ -62,7 +62,7 @@ private void EnableZoomOnBlendEnd_Internal(CinemachineVirtualCamera fromCamera,
 
     public ICinemachineCamera GetActiveCamera()
     {
-        return brain.ActiveVirtualCamera;
+        return brain == null ? null : brain.ActiveVirtualCamera as CinemachineVirtualCamera;
```

Comment:
brain looooool

Reply:
LOOOL yee they call it the CinemachineBrain cause it takes over the main camera :))))